corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -134,6 +135,17 @@ def update_sync_log_with_checks(sync_log, xform, cases, case_db,
                                         case_id_blacklist=case_id_blacklist)
 
 
+def prune_previous_log(sync_log):
```

Comment:
I think that means every submission for icds as they are trained to sync after every form.

Anyways not necessary to fix this. Looks like tests are failing in a weird way though (happened twice in a row)